### ISSUE TITLE
High-efficiency Variable Speed RTU (Replace Air System Cooling Equipment)

### ISSUE THREAD

**Issue by khaddad** [canmet-energy/btap_tasks#268]
### Description
Write a decription of the issue. If it is a bug, please list all steps and attach a sample file, or sample ruby code that exhibits the bug. 

### Approach
After reviewing the issue, decribe your solution. If including git hyperlinks, ensure hyperlinks to existing code are to a version so it does not change with updates to git. 

### Testing Plan
Enter testing plan, if you are going to use an existing test or make a new one. 

### Waiting On
If this task is dependent on another task or NREL. Include notes Here. 

### Repositories Involved

https://github.com/canmet-energy/btap-development-environment
https://github.com/canmet-energy/docker-openstudio
https://github.com/canmet-energy/btap
https://github.com/NREL/openstudio-standards/tree/nrcan

### Select the Project Code you will be charging to 2018-2019

| Client 	|WSBE  	|Name   	|Manager   	
|----------	|---	|---	|---	

|CEO  |P-004001.001 |Cost-Effective Pathways for NZE, NZE Ready, and GHG Neutral Buildings |Phylroy Lopez
